### PR TITLE
Issue/15 editor styling calypso

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Set the default style of the editor to match Calypso as implemented in the [Visual Editor](https://github.com/wordpress-mobile/WordPress-Android/tree/develop/libs/editor/WordPressEditor), as outlined in the [Design Handbook](https://wordpress.com/design-handbook/), and as described in https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/15.

### Test
Remove the following attributes from [`activity_main.xml`](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/app/src/main/res/layout/activity_main.xml) of the app
```
aztec:background
aztec:bulletColor
aztec:bulletMargin
aztec:bulletPadding
aztec:bulletWidth
aztec:historyEnable
aztec:historySize
aztec:lineSpacingExtra
aztec:lineSpacingMultiplier
aztec:linkColor
aztec:linkUnderline
aztec:quoteBackground
aztec:quoteColor
aztec:quoteMargin
aztec:quotePadding
aztec:quoteWidth
aztec:textColor
aztec:textColorHint
```

### CODE REVIEW

**Comment by 0nko** [comment]
Some extra padding around the `QuoteSpan` and withouth the vertical bar would probably make it look better (like in wpandroid) but that might not be easy to do...
         
image

**Comment by theck13** [comment]
Those three commits don't look related to the vertical bar, but they are.  I made the default color the same as the background and left the ability to change it later.

As far as the padding goes, I'm not sure if we can do that without being too hacky.  I'll look into it.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+    <!-- TOAST -->
+    <string name="toast_bold">Bold</string>
+    <string name="toast_bullet">Bullet</string>
+    <string name="toast_format_clear">Format clear</string>
+    <string name="toast_html">HTML</string>
+    <string name="toast_insert_link">Insert link</string>
+    <string name="toast_italic">Italic</string>
+    <string name="toast_quote">Quote</string>
+    <string name="toast_strikethrough">Strikethrough</string>
+    <string name="toast_underline">Underline</string>
```
Comment (0nko) [inline app/src/main/res/values/strings.xml]: I noticed these aren't used anywhere.

Comment (theck13) [inline app/src/main/res/values/strings.xml]: Good catch.  I missed those in the merge conflict resolution.

**Comment by 0nko** [comment]
Great! :shipit:

**Pull request merged**